go.mod

```diff
@@ -6,15 +6,16 @@ go 1.20
 replace sigs.k8s.io/controller-runtime => ./third_party/sigs.k8s.io/controller-runtime
 
 require (
-	cloud.google.com/go/trace v1.8.0
+	cloud.google.com/go/trace v1.9.0
 	contrib.go.opencensus.io/exporter/ocagent v0.7.0
 	contrib.go.opencensus.io/exporter/prometheus v0.4.2
 	contrib.go.opencensus.io/exporter/stackdriver v0.13.14
-	github.com/dapr/go-sdk v1.6.0
+	github.com/dapr/go-sdk v1.8.0
```

Comment:
We may need to start documenting dapr version compatibilities for each version of GK. wdyt?

Reply:
Yeah, I could add it to doc PR 🤔

Reply:
Do you have any specific place in mind for this information?

Reply:
For docs, might be good to add to the pubsub dapr provider docs.
For the daprClient, is there any minimum required version we need to test? Does the dapr sdk return its own version? might be good to include that in the log as the daprClient is created.

Reply:
As follow up when we implement batch publishing, `1.8.0` will be the minimum required version, and `dapr` 1.11 will be the minimum compatible version.

> Does the dapr sdk return its own version?

yes it does, but I am not sure how user can figure out which dapr runtime to install before hand if daprClient version information is going to be logged when user has installed all prerequisites and trying to initiate a connection using dapr.

Can we just say in the provider doc that, "current dapr sdk version is [link to gk go mod file]" and point to [dapr-version](https://docs.dapr.io/operations/support/support-release-policy/#supported-versions) for more information?